32-bit qrenderdoc build failure ("SWIG wrapped code invalid in 32 bit architecture, regenerate code using -DSWIGWORDSIZE32")

Building renderdoc 1.0 on both Mageia 6 i586 (GCC 5.5.0) and Mageia 7 i586 (GCC 7.3.0) (Linux), I get the following build error:
```
[100%] Building CXX object qrenderdoc/Code/pyrenderdoc/CMakeFiles/_qrenderdoc.dir/__/__/renderdoc_python.cxx.o
cd /home/iurt/rpmbuild/BUILD/renderdoc-1.0/build/qrenderdoc/Code/pyrenderdoc && /usr/bin/c++   -DDISTRIBUTION_CONTACT=\"https://madb.mageia.org/package/show/name/renderdoc\" -DDISTRIBUTION_NAME=\"Mageia\" -DDISTRIBUTION_VERSION=\"1.0-1.mga6\" -DGIT_COMMIT_HASH=\"5ef2d0bb5e86a37ba6c250987e9dc8c33c26435c\" -DRENDERDOC_PLATFORM_LINUX -DRENDERDOC_STABLE_BUILD=1 -DRENDERDOC_SUPPORT_GL -DRENDERDOC_SUPPORT_VULKAN -DRENDERDOC_WINDOWING_XCB -DRENDERDOC_WINDOWING_XLIB -D_qrenderdoc_EXPORTS -I/home/iurt/rpmbuild/BUILD/renderdoc-1.0/qrenderdoc -I/home/iurt/rpmbuild/BUILD/renderdoc-1.0/renderdoc/api/replay -I/usr/include/python3.5m  -O2 -g -pipe -Wformat -Werror=format-security -Wp,-D_FORTIFY_SOURCE=2 -fstack-protector --param=ssp-buffer-size=4 -fomit-frame-pointer -march=i586 -mtune=generic -fasynchronous-unwind-tables -std=c++11 -fstrict-aliasing -fvisibility=hidden -fvisibility-inlines-hidden -Wall -Wextra -Wno-unused-variable -Wno-unused-parameter -Wno-unused-result -Wno-type-limits -Wno-missing-field-initializers -Wno-unknown-pragmas -Wno-reorder -Wno-unused-but-set-variable -Wno-maybe-uninitialized -Werror -Wno-unknown-warning-option -Wno-unused-private-field -DNDEBUG -fPIC   -std=gnu++14 -o CMakeFiles/_qrenderdoc.dir/__/__/renderdoc_python.cxx.o -c /home/iurt/rpmbuild/BUILD/renderdoc-1.0/build/qrenderdoc/renderdoc_python.cxx
/home/iurt/rpmbuild/BUILD/renderdoc-1.0/build/qrenderdoc/renderdoc_python.cxx:4252:3: error: #error "SWIG wrapped code invalid in 32 bit architecture, regenerate code using -DSWIGWORDSIZE32"
 # error "SWIG wrapped code invalid in 32 bit architecture, regenerate code using -DSWIGWORDSIZE32"
   ^
```

Full build log: 
[build.0.20180309093408.log.txt](https://github.com/baldurk/renderdoc/files/1796318/build.0.20180309093408.log.txt)

--- a/qrenderdoc/CMakeLists.txt
+++ b/qrenderdoc/CMakeLists.txt
@@ -16,7 +16,11 @@ if(STATIC_QRENDERDOC)
 endif()
 
 if(NOT APPLE)
-    set(SWIG_FLAGS "-DSWIGWORDSIZE64")
+    if(CMAKE_SIZEOF_VOID_P EQUAL 8) # 64-bit
+        set(SWIG_FLAGS "-DSWIGWORDSIZE64")
+    else() # 32-bit
+        set(SWIG_FLAGS "-DSWIGWORDSIZE32")
+    endif()
 endif()
 
 if(QRENDERDOC_NO_CXX11_REGEX)

